backend/data_handling/data_processing.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
```

Comment:
Oh i didnt realize your backend was in Python, so it makes sense now why you wanted to use pandas 👍